tests/cdb2_admin.test/runit

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
The test can be vastly simplified. You just need the following to test if an admin connection is marked out in the connections system table.

```
cdb2sql --admin $dbname "SELECT SLEEP(10)" &
is_admin=`cdb2sql --tabs $dbname "SELECT is_admin FROM comdb2_connections WHERE pid=$!"
[ $is_admin != 0 ] && { exit 1; }
```

Reply:
Thanks Rivers! Will incorporate your suggestions